cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -325,12 +458,13 @@ void test_outlet_crud() {
             .statusCode(is(HttpServletResponse.SC_NOT_FOUND));
     }
 
-    private static Outlet buildTestOutlet(Location outletLoc, Location projectLoc, CwmsId ratingId) {
+    private static Outlet buildTestOutlet(Location outletLoc, Location projectLoc, CwmsId ratingId, String ratingSpecId) {
         return new Outlet.Builder().withProjectId(new CwmsId.Builder().withName(projectLoc.getName())
```

Comment:
Overload the ratingSpecId into this, rather than update stuff to use null.  Makes the code a bit more readable in the mess that's already there.

Reply:
Added overload, removed null fields in method calls